### PR TITLE
Update the university search page

### PR DESCRIPTION
**Pull request by superdev808**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=https://meettrax.com/api
```
Comment (tempoxylophone) [inline .env.example]: What is this API?

File: src/containers/Home/index.js [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
-import Spinner from "react-bootstrap/Spinner";
 import SearchInput from "../../components/SearchBox";
 
 export default function Home() {
   const [schools, setSchools] = useState([]);
   const [loading, setLoading] = useState(false);
-  const [total, setTotal] = useState(0);
-  const [pageNum, setPageNum] = useState(0);
+  const [total, setTotal] = useState(1);
+  const [pageNum, setPageNum] = useState(1);
```
Comment (tempoxylophone) [inline src/containers/Home/index.js:15]: Fine for now but in the future the backend should return pages

**Review by tempoxylophone — APPROVED** [review]

File: src/containers/Home/index.js (hunk 2 above)
Comment (superdev808) [reply src/containers/Home/index.js:15]: the page number will be updated like this.
```
fetchSchoolsList(queryString, pageNum, pageSize)
      .then((response) => {
        const { universities, total, page } = response;
        setSchools([...schools, ...universities]);
        setTotal(total);
        setPageNum(page);
      })
```

File: .env.example (hunk 1 above)
Comment (superdev808) [reply .env.example]: refactored with `transfercompass.com`

**Pull request merged**